rhizome/host/lib/boot_image.rb

```diff
@@ -77,6 +81,15 @@ def curl_image(url, temp_path, ca_path)
     sha256_sum
   end
 
+  def aria2_image(url, temp_path, sha256sum)
+    sha256_arg = sha256sum ? " --checksum=sha-256=#{sha256sum.shellescape}" : ""
```

Comment:
`sha256sum` should always be in hex, and therefore shouldn't need to be escaped. Not a problem if you want to keep the `shellescape` as a failsafe, though.